src/librustc_middle/query/mod.rs

```diff
@@ -91,6 +91,7 @@ rustc_queries! {
 
         /// Records the type of every item.
         query type_of(key: DefId) -> Ty<'tcx> {
+            desc { |tcx| "getting type of `{}`", tcx.def_path_str(key) }
```

Comment:
Not sure "getting" is the best wording (but also idk what previous discussion has happened on the topic).
I'd probably prefer "computing" instead.

Reply:
I've changed a few queries which are essentially caches in front of the HIR/crate metadata to "looking up", as "computing" didn't feel right.